cmd/geth/chaincmd.go

```diff
@@ -723,8 +723,12 @@ func downloadEra(ctx *cli.Context) error {
 	// Resolve the destination directory.
 	stack, _ := makeConfigNode(ctx)
 	defer stack.Close()
-	ancients := stack.ResolveAncient("chaindata", "")
+
+	ancients := stack.ResolveAncient("chaindata", "chain")
```

Comment:
`stack.ResolveAncient("chaindata", "chain")` will return `datadir/chaindata/chain` instead of `datadir/chaindata/ancient`

While in the description of Era flag

```go
	EraFlag = &flags.DirectoryFlag{
		Name:     "datadir.era",
		Usage:    "Root directory for era1 history (default = inside ancient/chain)",
		Category: flags.EthCategory,
	}
```

It says the default location is `datadir/chaindata/ancient/chain`